src/Core/Silk.NET.Core/Loader/LibraryLoader.cs

```diff
@@ -304,7 +304,7 @@ public void FreeNativeLibrary(IntPtr handle)
         public static LibraryLoader GetPlatformDefaultLoader()
         {
 #if NETCOREAPP3_0
```

Comment:
If we're upgrading to NETCOREAPP3_0, you'll need to replace all usage of this define too.

TFMs of `netstandard2.0;netcoreapp3.1` only define:
- NETSTANDARD
- NETSTANDARD2_0
- NETCOREAPP
- NETCOREAPP3_1

Reply:
Wait, now I'm realllly confused.
I thought Higher tfm means it defines everything below it. Ie net5.0 defines both NET5_0 and NETCOREAPP3_1 and everything below

Reply:
Just tested it, and you are right. for some reason NETCOREAPP3_1 is defined by net5.0 but NETCOREAPP3_0 isn't defined by NETCOREAPP3_1

Reply:
Yeah, it doesn't define prior versions, only the versions you define in the target framework.